lib/dor_indexing/indexers/releasable_indexer.rb

```diff
@@ -40,22 +40,26 @@ def tags
 
       def tags_from_collection
         parent_collections.each_with_object({}) do |collection, result|
-          Array(collection.administrative.releaseTags)
-            .select { |tag| tag.what == 'collection' }
+          collection_object_client = Dor::Services::Client.object(collection.externalIdentifier)
```

Comment:
As this is a gem, Dor::Services::Client is not going to be configured. We might handle this by passing in a configured Client. See how this is handled for a Workflow Client.

Reply:
@justinlittman I think this is ready for another look.